utils/python/tests/scripts_regression_tests.py

```diff
@@ -884,12 +884,12 @@ def test_bless_test_results(self):
         test_id = "%s-%s" % (self._baseline_name, CIME.utils.get_timestamp())
         self.simple_test(False, "%s -t %s" % (comparg, test_id))
 
-        # compare_test_results should detect the fail
+        # compare_test_results should run to completion
         cpr_cmd = "%s/compare_test_results -b %s -t %s 2>&1" % (TOOLS_DIR, self._baseline_name, test_id)
-        output = run_cmd_assert_result(self, cpr_cmd, expected_stat=1)
```

Comment:
run cmds in scripts_regression_tests should go through run_cmd_assert_result even if they are expected to pass. just remove the expected_stat argument so it defaults to 0.